hashing/te-tag-query-python/all_in_one_lib/all_in_one.py

```diff
@@ -0,0 +1,132 @@
+##!/usr/bin/env python
+
+"""
+A wrapper around multi-stage ThreatExchange operations.
+
+Includes simple matching and writing back. Useful for quickly validating new
+sources of ThreatExchange data. A possible template for a native
+implementation in your own architecture.
+
+This helper heavily relies on a config file to provide consistent behavior
+between stages, and a state file to store hashes.
+"""
+
+import argparse
+import os
+import os.path
+import pathlib
+import sys
+import typing as t
+
+import TE
+
+from .collab_config import CollaborationConfig
```

Comment:
I'm not sure how I feel about them overall - I used them because it helped me to avoid committing to a specific package structure until I figured out how it should be distributed (after which a consistent top-level package name will appear, like pytx3.collab_config or threatexchage.collab_config or facebook.threatexchange. etc)